### DISCUSSION TITLE
BCC-to-FCC transformation

### DISCUSSION THREAD

**Discussion by NovakEvana** [ORTools4MTEX/ORTools#34] [Q&A]
Dear Frank,

Long time no talk. I hope you are doing well.
I would like to have a quick post here to see your idea before I'm asking something further. I'm currently working with a BCC-based alloy. The material was uniaxially deformed at high velocity (strain rate of >3000s-1). Some suspicious things happened when we detected some FCC in our alloy after the deformation. Thus we suspect there are some BCC-to-FCC transformations that happened during the deformation process. Examination of some adjacent BCC and FCC grains shows similar locations of some poles on their corresponding pole figures based on K-S and N-W OR (please see images below). Nevertheless, to my knowledge, these ideal ORs and even your parent phase reconstruction/variant selection analysis are only applicable for transformations from FCC to BCC or FCC to HCP. Do you think that these rational ORs can also be employed for a BCC-to-FCC transformation? Otherwise, could you suggest any other mechanisms of transformation or other ORs that can be used here? Thank you.

![image](https://user-images.githubusercontent.com/117882416/224474393-4392d2f0-253c-40ea-8de7-e473d2659807.png)

Best Regards,
Evana.

**Comment by frankNiessen** [comment]
Hi Evana,

That sounds pretty unusual and interesting. Indeed ORTools is also aimed at OR discovery, you could for instance try the Peak fitting method as demonstrated in [example 2](https://github.com/ORTools4MTEX/ORTools/blob/develop/ORTools_example2.m) to find your OR.

Also the syntax in example 1

```
job.p2c = orientation.KurdjumovSachs(job.csParent, job.csChild);
misorientationsjob.calcParent2Child
```

will find the best fitting average OR based on an initial guess that is somewhat close to this OR, here K-S. Note that the real OR in microstructures often varies locally, so we can only determine a best fit, and that real ORs often are irrational.

I hope this helps,

Best wishes
Frank

**Reply by NovakEvana** [reply]
Thanks, Frank, I found it interesting too. I did the OR analysis today with a selection: the parent phase is BCC and the child phase is FCC. The peak fitting results are as follows. Do you think this is a good fit?

![image](https://user-images.githubusercontent.com/117882416/224568178-b9e16030-f813-414e-98d3-ea9f90e0d6a1.png)

**Reply by frankNiessen** [reply]
Hi Evana, 

it does not look too convincing unfortunately. I do not know your dataset, but it seems that there is a quite small volume fraction of the child phase present, as the plot looks quite noisy. 

You can see that the fitted OR still has a quite high disorientation (a good fit should give a low disorientation). Also K-S and N-W do not make too much sense when you change out the parent and the child phase from the standard convention. If you have a small dataset with few occurrences of the child phase, maybe consider looking for the boundary misorientation directly? You could plot the grain map and click the grains for getting the orientations of the neighboring grains, then you can calculate the misorientation. 

Best wishes
Frank
